### DISCUSSION TITLE
Imp Not Working

### DISCUSSION THREAD

**Discussion by FuadRamses** [chromiecraft/chromiecraft#5783] [General]
Hello,

I rolled as a warlock a little while ago, and my imp won't attack on coomand.

Also, I'm having to manually apply my imp's Blood Pact buff as well.

I tried signing off/logging back on, as well as dismissing/summoning the imp, but nothing's working. The imp still won't attack on command.

**Comment by 55Honey** [comment]
Hello @FuadRamses 
Please right click the spells you want your pet to cast automatically.

Regards.

**Reply by FuadRamses** [reply]
Thank you!
